### PR TITLE
Kuzbo-125 browse and create persisted documents

### PR DESCRIPTION
**Pull request by jenow**
- Can create new documents
- Refactored CRUDLDocument to move things from vuex inside the component
- Refactored Security/Users/List to be compatible with the new CRUDLDocument component

### CODE REVIEW

File: src/components/Data/Collections/Browse.vue [hunk 1]
```diff
@@ -1,13 +1,135 @@
...
+            sorting = filters.sort
+          }
+        }
+
+        if (this.sorting) {
+          sorting = formatSort(this.sorting)
+        }
+
+        // Execute search with corresponding filters
+        performSearch(this.$route.params.collection, this.$route.params.index, filters, pagination, sorting).then(res => {
```
Comment (stafyniaksacha) [inline src/components/Data/Collections/Browse.vue]: be carefull about usage of `this.$route.params` please use `collection` & `index` from properties

File: src/components/Data/Collections/Browse.vue [hunk 2]
```diff
@@ -1,13 +1,135 @@
 <template>
   <div class="wrapper">
-    <crudl-document :index="$route.params.index" :collection="$route.params.collection">
-      <document-item></document-item>
+    <headline>
+      {{$route.params.collection}} - Browse
```
Comment (stafyniaksacha) [inline src/components/Data/Collections/Browse.vue]: be carefull about usage of `this.$route.params` please use `collection` & `index` from properties

File: src/components/Data/Documents/Create.vue [hunk 3]
```diff
@@ -1,11 +1,84 @@
...
+    components: {
+      Headline,
+      JsonForm
+    },
+    methods: {
+      create () {
+        if (this.id) {
+          this.newDocument._id = this.id
+        }
+        kuzzle.dataCollectionFactory(this.$route.params.collection, this.$route.params.index).createDocument(this.newDocument, (err, res) => {
```
Comment (stafyniaksacha) [inline src/components/Data/Documents/Create.vue]: same as above about `this.$route.params`

**Comment by AnthonySendra** [comment]
:+1:

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**